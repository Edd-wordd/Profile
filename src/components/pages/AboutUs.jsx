import React from 'react'
import MainDisplay from '../../components/sections/MainDisplay'
import MissionStatement from '../../components/sections/MissionStatement'
import WhatWeDo from '../../components/sections/WhatWeDo'
import Resources from '../../components/sections/Resources'

function AboutUs() {
  return (
    <>
      <MainDisplay
        mobileViewTitle="The story of us!"
        mobileViewSubtitle="lets go get this"
        mainViewTitle="A guaranteed method to building your business!"
        mainViewSubtitle="-Letting you focus on what you do best"
        href={`https://images.unsplash.com/photo-1600049780189-60f39be1949f?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2070&q=80`}
      />
      <WhatWeDo />
      <MissionStatement />
      <Resources />
    </>
  )
}

export default AboutUs
