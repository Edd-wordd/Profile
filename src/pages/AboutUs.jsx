import React from 'react'
import Footer from '../components/Footer'
import WhoWeAre from '../components/WhoWeAre'
import MainDisplay from '../components/MainDisplay'
import MissionStatement from '../components/MissionStatement'
import TechUsed from '../components/TechUsed'
import GetInTouch from '../components/GetInTouch'
import WhatWeDo from '../components/WhatWeDo'

function AboutUs() {
  return (
    <>
      <MainDisplay
        mobileViewTitle="The story of us!"
        mobileViewSubtitle="lets go get this"
        mainViewTitle="The story of us!"
        mainViewSubtitle="lets go get this bitches"
        href={`https://images.unsplash.com/photo-1581090700227-1e37b190418e?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=2550&q=80`}
      />

      <WhoWeAre />
      <MissionStatement />
      <WhatWeDo />
      <TechUsed />
      <GetInTouch />
      <Footer />
    </>
  )
}

export default AboutUs
