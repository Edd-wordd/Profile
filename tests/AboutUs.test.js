import React from 'react'
import testComponent from './testComponent'
import AboutUs from '../src/components/pages/AboutUs'

testComponent(<AboutUs />, 'AboutUs')
