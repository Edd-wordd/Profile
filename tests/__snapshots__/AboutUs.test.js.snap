// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs should render AboutUs 1`] = `"Component"`;
